internal/config/config.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	"github.com/go-flutter-desktop/hover/internal/build"
+	"github.com/go-flutter-desktop/hover/internal/log"
+)
+
+// Config contains the parsed contents of hover.yaml
+type Config struct {
+	loaded    bool
+	Target    string
+	Branch    string
+	CachePath string `yaml:"cache-path"`
+	OpenGL    string
+	Docker    bool
+}
+
+var config = Config{}
+
+// GetConfig returns the working directory hover.yaml as a Config
+func GetConfig() Config {
+	if !config.loaded {
+		c, err := ReadConfigFile(filepath.Join(build.BuildPath, "hover.yaml"))
+		if err != nil {
+			log.Errorf("%v", err)
+			log.Errorf("This command should be run from the root of your Flutter project.")
+			os.Exit(1)
```

Comment:
Should not be log.Errorf and os.Exit.
Existing hover application that haven't been initialized with this version of hover will not work.
If the go/hover.yaml file is not found, then only a warning should be printed.